File: src/01-easy/1d-bush-fire/solution.m
#import <Foundation/Foundation.h>
int main(){
    int n; scanf("%d",&n);
    while(n--){
        char s[256]; scanf("%s",s);
        int d=0,j=0,l=(int)strlen(s);
        while(j<l){if(s[j]=='f'){d++;j+=3;}else j++;}
        printf("%d\n",d);
    }
    return 0;
}

File: src/01-easy/1d-bush-fire/solution.swift
import Foundation
let n = Int(readLine()!)!
for _ in 0..<n {
    let s = Array(readLine()!); var d=0; var j=0
    while j < s.count { if s[j] == "f" { d+=1; j+=3 } else { j+=1 } }
    print(d)
}
